### PR TITLE
feat: add additional group generators

### PR DESCRIPTION
**Pull request by yum0e**

### CODE REVIEW

File: group-generators/generators/ethereum-power-users-zk-badge-holders/index.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  GroupGenerator,
+} from "topics/group-generator";
+
+const generator: GroupGenerator = {
+  generationFrequency: GenerationFrequency.Once,
+
+  generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
+    const bigQueryProvider = new BigQueryProvider({network: SupportedNetwork.POLYGON});
+
+    const donateEventABI =
```
Comment (leosayous21) [inline group-generators/generators/ethereum-power-users-zk-badge-holders/index.ts:16]: why donate?

File: group-generators/helpers/providers/big-query/big-query.ts [hunk 2]
```diff
@@ -179,7 +191,55 @@ export default class BigQueryProvider {
...
+    return response[0].map(
+      (event) =>
+        iface.parseLog({
+          topics: event.topics,
+          data: event.data,
+        }).args as any as T
+    );
+  }
+
+  public async getBadges<T>({
```
Comment (leosayous21) [inline group-generators/helpers/providers/big-query/big-query.ts:213]: getSismoZKBadges ?

File: group-generators/generators/ethereum-power-users-zk-badge-holders/index.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { BigNumberish } from "ethers";
```
Comment (leosayous21) [inline group-generators/generators/ethereum-power-users-zk-badge-holders/index.ts:1]: Rename the file in "ethereum-power-users-polygon-zk-badge-holders" ?

File: group-generators/generators/masquerade-zk-badge-holders/index.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  GroupGenerator,
+} from "topics/group-generator";
+
+const generator: GroupGenerator = {
+  generationFrequency: GenerationFrequency.Once,
+
+  generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
+    const bigQueryProvider = new BigQueryProvider({network: SupportedNetwork.POLYGON});
+
+    const donateEventABI =
```
Comment (leosayous21) [inline group-generators/generators/masquerade-zk-badge-holders/index.ts:16]: same here

**Review by gabin54 — APPROVED** [review]

**Pull request merged**